### ISSUE TITLE
Android

### ISSUE THREAD

**Issue by nuno** [Nykz/ionic-capacitor-googlemaps#1]
Have you tested on android by chance?
I have black map on android, works  in the  browser.

**Comment by Nykz** [comment]
Yes I do and it’s working on Android nicely but not on my iPhone, I have
MacBook Pro m1

**Comment by Nykz** [comment]
Contact me via Facebook messenger
https://www.facebook.com/profile.php?id=100005738530768
